docs/TypeScript.md

```diff
@@ -997,7 +1086,7 @@ Notice: in the `onRequest` hook, request.body will always be null, because the b
 
 [src](../types/hooks.d.ts#L35)
 
-preParsing` is the second hook to be executed in the request lifecycle. The previous hook was `onRequest`, the next hook will be `preValidation`.
+preParsing` is the second hook to be executed in the request lifecycle. The previous hook was `onRequest`, the next hook will be`preValidation`.
```

Comment:
Looks to me like there are mismatched back ticks here in the original document. Not sure why this change also removes a space between "be" and the last inlined code snippet.